### PR TITLE
Add functionality to enable/disable captcha #2796

### LINKED ISSUE #2796: Admin Message Tab: Add "Captcha" column with checkbox
In the admin message at https://susi.ai/admin/messages tab please add a column with the option to activate or deactivate captcha.

Screenshot from 2019-08-06 23-47-24

### PR DESCRIPTION
**Pull request by fragm3**
Fixes #2796

Changes: 
- Add functionality to enable/disable captcha
- Fix API key destructuring

Demo Link: https://pr-2810-fossasia-susi-web-chat.surge.sh

Screenshots of the change: 

Screenshot 2019-08-10 at 9 58 13 PM

### CODE REVIEW

**Comment by fragm3** [comment]
@akshatnitd for the captcha to verify on the server-side, it should always work on the client side, how else will we verify user captcha response.

**Comment by fragm3** [comment]
@akshatnitd that would be quite redundant 😿 , I'll do it if you say so

**Comment by fragm3** [comment]
@akshatnitd cross clients implementation makes sense, will do it 😸

**Comment by fragm3** [comment]
@akshatnitd made the changes.

File: src/components/Auth/ResetPassword/index.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ class ResetPassword extends Component {
     location: PropTypes.object,
     actions: PropTypes.object,
     history: PropTypes.object,
+    isCaptchaEnabled: PropTypes.bool,
```
Comment (akshatnitd) [inline src/components/Auth/ResetPassword/index.js]: `isClientCaptchaEnabled`

**Comment by fragm3** [comment]
@akshatnitd done

**Comment by mariobehling** [comment]
Sorry, but I need to revert this. You have not actually solved the issue. You solved something else that is not related to the issue. Not sure, what is your idea here. Please read the issue again.

**Pull request merged**